xrviz/coord_setter.py

```diff
@@ -3,10 +3,20 @@
 
 
 class CoordSetter(SigSlot):
+    """
+    To set and reset the data coordinates.
 
+    It uses a `Cross Selector <https://panel.pyviz.org/reference/widgets/CrossSelector.html>`_
+    to display a list of simple and coordinate variables.
+    Simple variables (which are not data coordinates) are available on
+    left side and default coordinates are available on right side.
+    To set variables as coordinates, make selection on left side and click
+    ``>>``. Similarly making selection on right side and clicking ``<<``
+    will reset the coordinates. Other panes would update themselves
+    accordingly, in response to this change.
+    """
     def __init__(self, data):
-        """
-        To intialize
+        """ Initializes the CoordSetter object.
```

Comment:
Not useful

Reply:
Also, as a matter of definition, this initialises an **instance**. The word "object" is ambiguous in python, and you should not use it unless you are referring to `object` (which is an instance of `type`, i.e., a class).